apollo-router/src/query_planner/fetch.rs

```diff
@@ -454,4 +462,21 @@ impl FetchNode {
     pub(crate) fn operation_kind(&self) -> &OperationKind {
         &self.operation_kind
     }
+
+    pub(crate) fn extract_authorization_metadata(
+        &mut self,
+        schema: &apollo_compiler::Schema,
+        global_authorisation_cache_key: &CacheKeyMetadata,
+    ) {
+        let doc = Document::parse(&self.operation, "query.graphql");
```

Comment:
Does it mean we are reparsing something ?

Reply:
we are parsing the subgraph query, it was not parsed before